doc: update the readme.md file with latest UI

SumitMangrati opened the pull request:
the readme.md file has been updated with the latest ui image 

Resolves #4

SumitMangrati committed:
latest UI image updated

added the latest ui image on the readme.md file

SumitMangrati committed:
latest UI image updated

updated the latest ui image on the read.md file

malivinayak commented on README.md:
```diff
@@ -4,7 +4,9 @@ README.md
 
-![image](https://user-images.githubusercontent.com/66154908/236675164-d66403be-460f-43a2-b696-00cd0a1ea1d1.png)
+
+
```
can you delete the extra new line

malivinayak replied:
If not then its ok. As you are new to open source I will directly approve the changes and merge the PR

SumitMangrati replied:
okay ill work to remove the extra lines

malivinayak reviewed (changes requested):
Nicely Done, just a small change

SumitMangrati committed:
updated ui image

the readme file has been updated with the latest ui image and also the extra lines has been removed

malivinayak reviewed (approved):
Perfect

malivinayak commented:
@SumitMangrati you are working on issue #4 

So in the PR description simple mention `Resolves #4` at the end
This will join this PR to that issue.

and after merging this PR, respective issue will automatically close

Try editing the description and I will merge the PR

malivinayak commented:
> @SumitMangrati you are working on issue #4
> 
> So in the PR description simple mention `Resolves #4` at the end This will join this PR to that issue.
> 
> and after merging this PR, respective issue will automatically close
> 
> Try editing the description and I will merge the PR

I am merging the PR
Just remember that you can link issue to PR y mentioning `Resolves #[issue No.]`

malivinayak commented:
@SumitMangrati You are trying to change title
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/990b4296-27ad-468a-989f-b5de50050cfd)


and I am talking about description
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/c333f4c1-d78f-4ebc-90b4-f28887fcdbed)


I have updated it just now

SumitMangrati committed:
latest UI image updated

SumitMangrati commented:
could you check it once i have worked on removing the extra lines

malivinayak commented:
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/cef0aae5-d306-44a8-934b-deac97eff98b)

Here you can see that after adding `Resolves #4` it automatically linked the issue

malivinayak commented:
> could you check it once i have worked on removing the extra lines

Yes, you have done it perfectly

SumitMangrati commented:
thank you now i understood by mentioning the number my pr gets directly linked to that particular issue

malivinayak commented:
> thank you now i understood by mentioning the number my pr gets directly linked to that particular issue

Yes.
I will merge the PR
Hope you learn something new😊

SumitMangrati commented:
Thank you so much I learned a lot from you. Started my journey of open source with you. In the Hacktober tab it says my PR needs to under 7 days reviews. Thank you

malivinayak commented:
> Thank you so much I learned a lot from you. Started my journey of open source with you. In the Hacktober tab it says my PR needs to under 7 days reviews. Thank you

Yes after 7 days it will automatically goes to accepted. It is one of the rule of hacktoberfest
They wait for 7 days to check if repo maintainers keep the PR or remove.

Don't worry about this PR. I have accepted it

malivinayak commented:
You can connect me at [LinkedIn](https://www.linkedin.com/in/malivinayak/)
